src/amber/cli/recipes/recipe.cr

```diff
@@ -57,7 +57,7 @@ module Amber::Recipes
           app = App.new(name, options.d, options.t, options.m, recipe)
           app.fetch_recipe(directory)
           app.render(directory, list: true, color: true)
-          if options.deps?
```

Comment:
positive checks are more readable, no?

Reply:
`if !option.no_deps?` is a double negative which I find harder to read myself.  This is one I think its acceptable to use the `unless`